src/main/java/com/commercehub/gradle/plugin/avro/GenerateAvroJavaTask.java

```diff
@@ -256,11 +256,11 @@ private void processSchemaFile(ProcessingState processingState, FileState fileSt
     }
 
     private void compile(Protocol protocol, File sourceFile) throws IOException {
-        compile(new SpecificCompiler(protocol), sourceFile);
+        compile(new SpecificCompiler(protocol, SpecificCompiler.DateTimeLogicalTypeImplementation.JSR310), sourceFile);
     }
 
     private void compile(Schema schema, File sourceFile) throws IOException {
-        compile(new SpecificCompiler(schema), sourceFile);
+        compile(new SpecificCompiler(schema, SpecificCompiler.DateTimeLogicalTypeImplementation.JSR310), sourceFile);
```

Comment:
I've specified JSR310 here. However, this should be configurable of course, and should probably default to `.DEFAULT` (which is JODA), or `.JODA`. The configuration mechanism of this plugin was a bit opaque so I didn't touch it. I noted it as a TODO in the pull request.

Reply:
Edit: The default is now JSR310: https://github.com/apache/avro/commit/ff6a4e8317689928056ed18805a944057ae666ad

Reply:
Thanks for the walkthrough. I've implemented this. See one comment on the code below.